docker/ftest-container-star-operator/pom.xml

```diff
@@ -68,6 +68,12 @@
       <artifactId>wildfly-arquillian-container-remote</artifactId>
       <version>${version.wildfly}</version>
       <scope>test</scope>
+      <exclusions>
```

Comment:
This is not required, and causes a build failure on OpenJDK

Reply:
I didn't get your point?

Reply:
The exclusion

Reply:
this = jconsole is not required

Reply:
jconsole = causes a build failure on OpenJDK

Reply:
So exluding it means that the build passes on OpenJDK